src/controllers/auth/addserver.js

```diff
@@ -46,9 +46,7 @@ define(["appSettings", "loading", "browser", "emby-button"], function(appSetting
         view.querySelector(".addServerForm").addEventListener("submit", onServerSubmit);
         view.querySelector(".btnCancel").addEventListener("click", goBack);
 
-        require(["autoFocuser"], function (autoFocuser) {
-            autoFocuser.autoFocus(view);
-        });
+        view.querySelector("#txtServerHost").focus();
```

Comment:
In the current case, there is no harm - scrolling is not required. But `scrollManager` expects `preventScroll: true`. So `focusManager.focus(elem)` is preferred.